core/src/main/scala/org/apache/spark/util/logging/FileAppender.scala

```diff
@@ -76,7 +76,11 @@ private[spark] class FileAppender(inputStream: InputStream, file: File, bufferSi
           }
         }
       } {
-        closeFile()
+        try {
+          inputStream.close()
```

Comment:
AFAIK this inputStream is passed from outside, like `process.getInputStream` in `ExecutorRunner`, so I don't think we should close the inputStream here.